Lib/test/test_socket.py

```diff
@@ -1519,6 +1519,24 @@ def test_unusable_closed_socketio(self):
             self.assertRaises(ValueError, fp.writable)
             self.assertRaises(ValueError, fp.seekable)
 
+    def test_socket_close(self):
```

Comment:
Is there already a test for `dup()`?

Reply:
Ping on this question.

Reply:
No, there is no test. I don't have time to write a test case, Would you like me to drop the documentation for dup?